### ISSUE TITLE
Cannot disable ctrl and shift keys to prevent select?

### ISSUE THREAD

**Issue by gokanovic** [d3lm/ngx-drag-to-select#165]
I want to disable the ctrl and shift key to prevent select. As far as I know that is not possible or is it?

**Comment by d3lm** [comment]
How do you mean ctrl + shift, that shortcut is not implemented, see 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/12571019/201919517-53adddd3-bab2-4434-8535-5d32a5cdc300.png">

I am afraid that without a reproducable and a list of steps there's not much I can do.

**Comment by gokanovic** [comment]
I just want disable ctrl key and shift key. I don't want that the user is going to use these keys to make a selection. I want a single selection mode with no use  of ctrl or shift keys

**Comment by d3lm** [comment]
Have you checked the live page and played with the options? This does exactly what you want IIUC
<img width="592" alt="image" src="https://user-images.githubusercontent.com/12571019/202682876-a0e973ac-00f5-4dbb-94c0-f686431f1b98.png">

**Comment by gokanovic** [comment]
![image](https://user-images.githubusercontent.com/741560/202687326-0ed74a4b-1797-4814-9e25-25c63c0b3cd1.png)

I think you really don't understand me. In live demo, if you uncheck "select mode" there, you can still make multiple selections by holding the ctrl key together. And that's what I don't want. I want to turn this off. As far as I can see that is not possible.

**Comment by d3lm** [comment]
Yep, that is currently not supported. With CTRL or CMD (on Mac) you toggle a selection. There is currently not a single selection mode.
